.pre-commit-config.yaml

```diff
@@ -14,13 +14,24 @@ repos:
   - id: debug-statements
   - id: forbid-new-submodules
 - repo: https://github.com/psf/black
-  rev: stable
+  rev: 19.10b0
   hooks:
   - id: black
 - repo: https://gitlab.com/pycqa/flake8
-  rev: 3.8.1
+  rev: "3.8.1"
```

Comment:
Why is this one in quotes when the others aren't? I find YAML syntax fiddly enough that I'd prefer consistency.

Reply:
Because YAML is picky... and dumb at the same time. If something starts with numbers and then a ``.`` it thinks it's a floating number. Others start with ``v`` to avoid this issue.

Reply:
OK, so let's quote them all for consistency, I guess :-(

But don't do that yet, I'll check what other projects do and come to a decision on what style I like. All options seem to suck, unfortunately.

Reply:
Well, I did not change it. Rather ``pre-commit autoupgrade`` did. The ``pre-commit`` policy is to use the non-quoted version where it's needed to remove ambiguity (as is here), but keep the short version otherwise where it's needed as it's more readable. IMHO this is a good enough policy to me. This file, in general, is not meant to be human maintained, but rather by ``pre-commit`` itself once the checks have been added. I don't think ``pre-commit`` has an ``always-quote`` option, @asottile?

Reply:
OK, so maybe we don't bother with that helper for now, we can do things manually in the short term. If checking what the tool did is more effort than not using the tool, I'm fine with holding off on using the tool for now.

Reply:
Why do you care so strongly about how a tool generates a configuration files content as long as that is done automatically?

Reply:
Maybe I don't :-) I'll have a think - I'm used to manually editing at least some of this stuff, so maybe I need to just embrace a new approach :-)

Reply:
huh, it shouldn't have quoted this one -- must be the feature that someone just added let me check that 🤔

Reply:
doesn't quote it for me 🤷  -- but as of pre-commit 2.4 if you quote them then pre-commit will keep them quoted after autoupdate